sources/packages/web/src/components/generic/StatusChipBypass.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <chip-status :status="chipStatus" :label="bypassStatus" />
+</template>
+<script lang="ts">
+import { computed, defineComponent, PropType } from "vue";
+import ChipStatus from "@/components/generic/ChipStatus.vue";
+import { ApplicationRestrictionBypassStatus } from "@/services/http/dto";
+import { useBypassStatus } from "@/composables";
+
+export default defineComponent({
+  components: { ChipStatus },
+  props: {
+    isActive: {
+      type: Boolean as PropType<boolean>,
+      required: true,
+    },
+  },
+  setup(props) {
+    const { mapBypassStatus } = useBypassStatus();
+    const bypassStatus = props.isActive
+      ? ApplicationRestrictionBypassStatus.Active
+      : ApplicationRestrictionBypassStatus.Removed;
```

Comment:
Please use a computed property here. Please use the reference code here [Computed Example](https://github.com/bcgov/SIMS/blob/a4c92db1bef85a1f217a25f198562537805ec791/sources/packages/web/src/components/generic/StatusChipProgram.vue#L24)

Reply:
Updated. `bypassStatus` is now reactive to the parent component with `computed()`.